src/Features/Core/Portable/ChangeSignature/AbstractChangeSignatureService.cs

```diff
@@ -807,16 +808,64 @@ protected static int GetParameterIndex<TNode>(SeparatedSyntaxList<TNode> paramet
                             continue;
                         }
 
+                        SyntaxNode expression = null;
+
+                        var callsiteInferenceFailed = false;
+
+                        if (addedParameter.CallSiteKind == CallSiteKind.Inferred && addedParameter.TypeBinds && getSemanticModelAndRecommendations != null)
+                        {
+                            callsiteInferenceFailed = true;
+
+                            (var semanticModel, var recommendations) = await getSemanticModelAndRecommendations().ConfigureAwait(false);
```

Comment:
yeah, def not seeing hte value of this function.

Reply:
Removed indirection.